client/src/app/components/home/home.component.ts

```diff
@@ -1,20 +1,46 @@
-import { Component, OnInit } from '@angular/core';
+import { MediaMatcher } from '@angular/cdk/layout';
+import { AfterViewInit, ChangeDetectorRef, Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { MatSidenav } from '@angular/material/sidenav';
 import { TokenService } from 'src/app/services/token.service';
 import { Router } from '@angular/router';
+import { SidenavService } from 'src/app/services/sidenav.service';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.css'],
+  providers: [SidenavService],
```

Comment:
I would say don't add the SidnavService inside the home component, do it inside the Alta-main-module.

Reply:
Yes nice catch. This is actually not needed at all. I removed it with 6589be7856d36674cec240568ad3eb202f7d177d

Reply:
But you forgot to add it to the alta-main-module inside the providers decorator.

Reply:
Yes because its not needed there as well